docs/math.md

```diff
@@ -69,3 +69,10 @@ Returns the greatest float value greater than or equal to input value
 Returns a float value with the remainder rounded to the given number to digits after the decimal point.
 
 `round 123.555555 3` will return `123.556`
+
+## randInt
+Returns a random integer value from min (inclusive) to max (exclusive).
+```
+{{ randInt 12 30 }}
+```
+The above will produce a random number in the range [12,30).
```

Comment:
```suggestion
The above will produce a random number in the range [12,30].
```

Reply:
That notation is correct: https://en.wikipedia.org/wiki/Interval_(mathematics)#Integer_intervals

Reply:
TIL